io.github.otaxhu.MQTTy.yml

```
@@ -0,0 +1,52 @@
+---
+
+id: io.github.otaxhu.MQTTy
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+command: MQTTy
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+
+modules:
+  - name: blueprint-compiler
+    buildsystem: meson
+    cleanup:
+      - "*"
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/jwestman/blueprint-compiler
+        tag: v0.16.0
+
+  - name: sass
+    buildsystem: simple
+    cleanup:
+      - "*"
+    build-commands:
+      - cp -r * ${FLATPAK_DIST}/bin
+      - chmod +x ${FLATPAK_DIST}/bin/sass
```

```suggestion
      - cp -r * ${FLATPAK_DEST}/bin
      - chmod +x ${FLATPAK_DEST}/bin/sass
```

It ain't gonna work otherwise.

---

That's the build failure, btw.